.github/ISSUE_TEMPLATE/use-case.yml

```diff
@@ -0,0 +1,25 @@
+name: FAIR Mappings Use Case submission
+description: Use this template to submit a FAIR Mappings Use Case for the RDA FAIR Mappings Working Group.
+title: FAIR Mappings Use Case submission
+labels: [ use-case ]
+assignees: jkesanie
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some instructions for the submission.
+  - type: input
+    id: use-case-title
+    attributes:
+      label: The title of your use case
+      description: A short but descriptive title that describes your use case.
+    validations:
+      required: true
+  - type: textarea
+    id: comments
+    attributes:
+      label: Additional comments
+      description: |
+        Please provide any additional comments here.
```

Comment:
Bike-shedding: I would say “please provide any additional **details** here”. IMO, “comments” suggests that this is entirely superfluous, and would encourage not giving any comments at all.

Reply:
It was not my intention to create a field here that will actually be used - only an example of a use of a `textarea` for @jkesanie to create his survey